test/expression.test.js

```diff
@@ -49,6 +49,8 @@ run('js', {ignores, tests}, (fixture) => {
         for (const input of fixture.inputs || []) {
             try {
                 const feature = {properties: input[1].properties || {}};
+                availableImages = input[0].availableImages;
```

Comment:
nit: `availableImages = input[0].availableImages || [];` ?

Reply:
I had that originally but the tests all passed without it. I can add it back just to be defensive though